### PR TITLE
fix: small adjustment request 

### LINKED ISSUE #54: [Demand] Mask.io official website - small adjustment request
## Catalogue

- [x] UI/UX
- [ ] Mask Core Feature
- [ ] Mask Plugin Feature
- [ ] Security/Cryptography Feature
- [ ] Other
  - /_List it here_/  Official website

## Part 2: Problem to Solve
One year after the release of Chromium Edge (Windows 10), complete support has been added to Chrome web store, now through simple steps one can get Mask plug-in directly in Chrome web store.
## Part 3: Expected Behavior
Adjustment of current website text content (small change)
Adjustment of the design of website text and picture content
## Notes
Mask-Pic

### PR DESCRIPTION
**Pull request by lelenei**
closes #54

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -100,9 +100,8 @@ <h1 class="headline">
             Get Mask
           </a>
           <span class="link-description">
-            For <span title="Google Chrome">Chrome</span>, <span title="Microsoft Edge">Edge</span>, <span
-              title="Mozilla Firefox">Firefox</span>,<br />
-            and Chromium-based browsers.
+            For <span title="Google Chrome">Chrome</span>, <span title="IOS">IOS & Android</span>,<br />
```
Comment (septs) [inline index.html:103]:
```suggestion
            for <span title="Google Chrome">Chrome</span>, <span title="iOS">iOS & Android</span>,<br />
```

Comment (guanbinrui) [reply index.html:103]: fix: typo

File: index.html [hunk 2]
```diff
@@ -100,9 +100,8 @@ <h1 class="headline">
             Get Mask
           </a>
           <span class="link-description">
-            For <span title="Google Chrome">Chrome</span>, <span title="Microsoft Edge">Edge</span>, <span
-              title="Mozilla Firefox">Firefox</span>,<br />
-            and Chromium-based browsers.
+            for <span title="Google Chrome">Chrome</span>, <span title="iOS">iOS & Android</span>,<br />
```
Comment (guanbinrui) [inline index.html:103]:
```suggestion
            For <span title="Google Chrome">Chrome</span>, <span title="iOS">iOS & Android</span>,<br />
```

**Review by guanbinrui — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -100,9 +100,8 @@ <h1 class="headline">
             Get Mask
           </a>
           <span class="link-description">
-            For <span title="Google Chrome">Chrome</span>, <span title="Microsoft Edge">Edge</span>, <span
-              title="Mozilla Firefox">Firefox</span>,<br />
-            and Chromium-based browsers.
+            For <span title="Google Chrome">Chrome</span>, <span title="iOS">iOS & Android</span>,<br />
+            and Chromium-based browsers E.g.,<span title="Microsoft Edge">Edge</span>
```
Comment (guanbinrui) [inline index.html:104]:
```suggestion
            and Chromium-based browsers, E.g., <span title="Microsoft Edge">Edge</span>.
```

**Pull request merged**